docs/handbook/image-file-formats.rst

```diff
@@ -1488,7 +1488,9 @@ QOI
 
 .. versionadded:: 9.5.0
 
-Pillow identifies and reads images in Quite OK Image format.
+Pillow reads images in Quite OK Image format, using a Python decoder. If you wish to
+write code specifically for this format, https://pypi.org/project/qoi is an alternative
+library that interfaces with NumPy.
```

Comment:
Mention C somehow, so they know it's faster?

```suggestion
library written in C that interfaces with NumPy.
```

Reply:
"written in C" sounds to me like the entire library is in C. How about "that uses C to decode the image"?

Reply:
Yeah, something like that's fine too.

Reply:
Cool, I've pushed a commit for that.